sig-list.md

```diff
@@ -66,6 +66,7 @@ When the need arises, a [new SIG can be created](sig-creation-procedure.md)
 |[Kubeadm Adoption](wg-kubeadm-adoption/README.md)|* [Lucas Käldström](https://github.com/luxas), Luxas Labs (occasionally contracting for Weaveworks)<br>* [Justin Santa Barbara](https://github.com/justinsb)<br>|* [Slack](https://kubernetes.slack.com/messages/sig-cluster-lifecycle)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)|* Regular WG Meeting: [Tuesdays at 18:00 UTC (bi-weekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
 |[Machine Learning](wg-machine-learning/README.md)|* [Vishnu Kannan](https://github.com/vishh), Google<br>* [Kenneth Owens](https://github.com/kow3ns), Google<br>* [Balaji Subramaniam](https://github.com/balajismaniam), Intel<br>* [Connor Doyle](https://github.com/ConnorDoyle), Intel<br>|* [Slack](https://kubernetes.slack.com/messages/wg-machine-learning)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-wg-machine-learning)|* Regular WG Meeting: [Thursdays at 13:00 PT (Pacific Time) (biweekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
 |[Multitenancy](wg-multitenancy/README.md)|* [David Oppenheimer](https://github.com/davidopp), Google<br>* [Jessie Frazelle](https://github.com/jessfraz), Microsoft<br>|* [Slack](https://kubernetes.slack.com/messages/wg-multitenancy)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-wg-multitenancy)|* Regular WG Meeting: [Wednesdays at 11:00 PT (Pacific Time) (biweekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
+|[Network Service Mesh](wg-network-service-mesh/README.md)|* [Ed Warnicke](https://github.com/edwarnicke), Cisco Systems<br>* [Frederick Kautz](https://github.com/fkautz), RedHat<br>* [Kyle Mestery](https://github.com/mestery), CIsco Systems<br>|* [Slack](https://kubernetes.slack.com/messages/)<br>* [Mailing List](https://groups.google.com/forum/#!forum/networkservicemesh)|* Regular WG Meeting: [Fridays at 08:00 PDT (weekly)](https://docs.google.com/document/d/1C9NKjo0PWNWypROEO9-Y6haw5h9Xmurvl14SXpciz2Y/edit#heading=h.rc9df0a6n3ng)<br>
```

Comment:
Others use PT, can we use PT here?

I personally prefer PST/PDT since it is more accurate, but since the rest use PT... best to use that.

Reply:
@cblecker k8s-network subproject vs wg is an excellent question.  We went to the k8-network meeting and asked it at their last meeting.  They (including @thockin)  recommended a WG.  We've been working with sig-network closely, and also with the wg-iot-edge (as they have networking needs they think we can solve).